app/controllers/any_login/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 module AnyLogin
   class ApplicationController < ActionController::Base
     include AnyLogin.provider::Controller
+    include ::Clearance::Controller if AnyLogin.provider.to_s.eql?("AnyLogin::Provider::Clearance")
```

Comment:
This being loaded the way it was before causes problems in the test suite. This is meant to be a temporary workaround until https://github.com/igorkasyanchuk/any_login/pull/47 is merged which changes this again.